test/unit/app/utils/test_utils_credit_transfer.py

```diff
@@ -0,0 +1,9 @@
+from server.utils import credit_transfer as CreditTransferUtils
```

Comment:
adding some basic unit tests for `test_utils_credit_transfer.py`. Need to get codecov relative paths working to check which lines are covered